Dockerfile-Dev

```diff
@@ -1,18 +1,5 @@
-FROM node:10
-
-# Install global packages
-RUN npm install -g gulp-cli mocha
-
-# Clone Habitica repo and install dependencies
-RUN mkdir -p /usr/src/habitrpg
-WORKDIR /usr/src/habitrpg
-RUN git clone https://github.com/HabitRPG/habitica.git /usr/src/habitrpg
-RUN cp config.json.example config.json

```

Comment:
Is using custom config values in config.json still supported?

Reply:
Sure. All the source code, including the `config.json`, are being shared from the local source code to the containers.
The only thing that is not synced in real time is the `node_modules` folder. If there are new packages, we should rebuild the docker image.